### PR TITLE
Add announcements

### PR DESCRIPTION
**Pull request by Prajjwal**

### CODE REVIEW

File: app/pods/classroom/timeline/announcements/template.hbs [hunk 1]
```diff
@@ -1,155 +1,50 @@
...
-                We create a simple design flow using Sketch!
-                We put our prototype on Marvel
-            </div>
-
-
-        </div>
+		Announcements ({{run.announcements.length}})
     </div>
+	{{#each run.announcements as |announcement|}}
+		<div class="accordian-ul" data-show="true">
```
[the remark this answers is not in the document]
Comment (Prajjwal) [reply app/pods/classroom/timeline/announcements/template.hbs]: Can't find the component.

[the remark this answers is not in the document]
Comment (Prajjwal) [reply app/pods/classroom/timeline/announcements/template.hbs]: Never mind, found it. 'Accordion' is misspelled everywhere.

**Comment by abhishek97** [comment]
This is still static
screen shot 2018-05-08 at 4 46 40 pm

Author Name is still hardcoded. Also the accodion "-" doesn't toggles to "+"
screen shot 2018-05-08 at 4 48 53 pm

The `load more` button doesn't work. No action is attached to it.

**Comment by witty123** [comment]
@Prajjwal can you please do the required changes ? Its been over a month and a half since this feature was assigned to you.

**Comment by Prajjwal** [comment]
@witty123 Changes were requested 23 hours ago. I'm doing them.

**Pull request merged**